cmd/scsd/creds.go

```diff
@@ -1112,3 +1130,49 @@ func doCredsGet(w http.ResponseWriter, r *http.Request) {
 	w.Write(ba)
 }
 
+func doHSMDiscover(xnames []string) ([]byte, error) {
+	if len(xnames) == 0 {
+		return nil, nil
+	}
+	logger.Error("********** IN HSM DISCOVER ****************")
```

Comment:
Maybe we can change this log message to be more informational? Something like:
```
logger.Println("Informing HSM to discover the following redfish endpoints:, strings.Join(xnames, ",")
```

Reply:
Removed